docs/TRANSLATING.md

```diff
@@ -1,80 +1,289 @@
+your attention — here the writing script is not specified. Most likely this is
+going to be a Latin script — we need to put it into the `msgstr` and remove the
+`fuzzy` flag, like this:
+
+```po
+#. Do not translate this, set according to the instruction!
+#: #METADATA
+msgctxt "#METADATA"
+msgid ""
+"#SCRIPT\n"
+"\n"
+"Writing script used in this language, can be one of:\n"
+"Latin, Arabic, Armenian, Cherokee, Cyrillic, Georgian, Greek, Hebrew"
+msgstr "Latin"
+```
 
-## Updating the translation
+> **Notes**
+>
+> Never ever change any other flags — `fuzzy` flag is the only one you are
+> allowed to remove (or add).
+>
+> Some dedicated `PO` editors use a different name for the `fuzzy` flag in
+> their user interfaces, like _translation status_, _work needed_, etc.
 
-Update a given translation by editing the corresponding `*.lng` file with a text
-editor and saving it with UTF-8 encoding and Unix line endings.
+### Translatable entries
 
-Suitable editors are often called ASCII editors, code editors, etc. —
-Microsoft Word is not suitable, but Notepad (Windows), Notepad++ (Windows),
-TextMate (macOS), or Kate (Linux) will do.
+Here is an example translatable entry:
 
-Please don't use tab characters — configure your editor to expand tabs to
-spaces. If you use tabs, there is a danger that the alignment of indented
-multi-line text will appear wrong.
+```po
+#: src/dos/programs/mount_common.cpp:76
+#, fuzzy, c-format, no-wrap
+msgctxt "PROGRAM_MOUNT_STATUS_1"
+msgid "The currently mounted drives are:"
+msgstr "Les lecteurs montés actuellement sont :"
+```
 
-Tools exist (often called diff tools) that can help you to compare two `en.lng`
-files generated from different versions of DOSBox, they make it easy to figure out
-what has changed in the English strings and what needs to be addressed in your
-updated translations. These include Meld (cross-platform), WinMerge (Windows),
-and Kompare (Linux).
+The location (after `#:`) tells where the message is actually located in the
+source code — it is updated every time you write the translation file.
+Some `PO` file editors can open the relevant source code if you configure the
+root path to the project.
+
+The `msgctxt` (a context according to gettext terminology) is a unique message
+identifier used by DOSBox Staging. Never ever change it, or the emulator won't
+be able to use the entry!
+
+The `msgid` contains an English message — as above, don't change it!
+
+The `msgstr` contains a translated message, empty if the translation is missing.
+This is the part you need to update :)
+
+Last but not least, the `fuzzy` flag we have already seen — it is set by DOSBox
+Staging to tell you this particular entry needs your attention. This might be
+a missing translation, an outdated translation (it is detected by comparing
+`msgid` to the current English message in the code), or any other reason — see
+the warning logs when loading the translation. After updating the translation,
+remove it!
```

Comment:
"After having updated the translation, remove the `fuzzy` flag!"

Reply:
OK, changed.